### PR TITLE
Make option to disable module visible in GUI

### PR DESCRIPTION
**Pull request by horstle**
As stated in #32 with this change you can enable/disable inputstreamhelper via GUI at 
"kodi settings" > "system" (change from standard to expert there) > "addons" > "manage dependencies" > "Inputstream Helper" > "configure".

### CODE REVIEW

**Comment by horstle** [comment]
I guess breaking compatibility with Kodi versions before Leia is not wanted (yet)? If you want to I can revert the second commit, doesn't change the functionality anyway.

**Comment by dagwieers** [comment]
I'd like to postpone this for the v0.5.0 release as this has an impact on how the plugin behaves.

One concern I have is that users do not have an friendly way to access this plugin's settings.
So this would then only be useful if (consumer) add-on provide a settings entry to access this one's.
And that will only be useful if the settings has a wider use.

PS Can you please rebase your PR?

**Comment by horstle** [comment]
Done.

File: lib/inputstreamhelper.py [hunk 1]
```diff
@@ -181,12 +181,8 @@ def _cmd_exists(cmd):
 
     def _helper_disabled(self):
         """Return if inputstreamhelper has been disabled in settings.xml."""
-        disabled = ADDON.getSetting('disabled')
-        if not disabled:
-            ADDON.setSetting('disabled', 'false')  # create default entry
-            disabled = 'false'
-
-        if disabled == 'true':
+        disabled = ADDON.getSettingBool('disabled')
```
Comment (vlmaksime) [inline lib/inputstreamhelper.py]: This method has been available since Kodi 18. It's broke backwards compatibility with Kodi 17 and later.

Comment (horstle) [reply lib/inputstreamhelper.py]: I asked whether I should revert this change before.
https://github.com/emilsvennesson/script.module.inputstreamhelper/pull/34#issuecomment-475909581

Comment (vlmaksime) [reply lib/inputstreamhelper.py]: > I'll have to create a new commit changing this back, if that's what you want.
Yes. I meant that. Thanks.

Comment (horstle) [reply lib/inputstreamhelper.py]: Reverted it.

**Comment by horstle** [comment]
OK, the commits have been really fucked up by the rebase, but the end result is fine. I think there's no harm in merging this and building a full-fledged setting page based on it.

**Comment by dagwieers** [comment]
I agree, this doesn't require any API calls. Let's get you out of this misery ;-)

**Comment by horstle** [comment]
Thanks

**Comment by dagwieers** [comment]
While reworking the interface for the future, I think we should redesign this interface.
Interfaces should list functionality, and functionality can be enabled or disabled.

The problem here is that "disabled" is actually a negative imposition. The value "true" should be enabled, and a value "false" should be disabled. And by default this functionality is enabled. This makes it a much better interface for users.

The problem with changing this is two-fold, either we support people that have used this manually and make it more complex and error-prone, or we don't, and those advanced users would need to reapply. IMHO I don't think we should be adding complexity, and expect that users do not have it disabled. (Especially with the recent Widevine CDM breakage, users have it enabled again ;-))

I will propose this in a subsequent PR.

**Pull request merged**